assets/global.js

```diff
@@ -308,17 +308,19 @@ class MenuDrawer extends HTMLElement {
     const detailsElement = summaryElement.parentNode;
     const isOpen = detailsElement.hasAttribute('open');
 
+    function addTrapFocus(event) {
```

Comment:
```suggestion
    function addTrapFocus() {
```

Event isn't being used here.